Description: Fix crashes due to undeclared functions
 The consequence of at least one of these (in geom_getxml.c) was a crash
 when there was a lot of data in kern.geom.confdml.
 .
 freebsd-libs (9.0+ds1-3.1) UNRELEASED; urgency=low
 .
   * Fix 'warning: implicit declaration of function 'reallocf'
     (and several other functions) (Closes: 685625)
Author: Jeff Epler <jepler@unpythonic.net>
Bug-Debian: http://bugs.debian.org/685625

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Forwarded: no
Last-Update: <2012-12-21>

--- freebsd-libs-9.0+ds1.orig/lib/libdevstat/devstat.c
+++ freebsd-libs-9.0+ds1/lib/libdevstat/devstat.c
@@ -34,6 +34,7 @@ __FBSDID("$FreeBSD$");
 #include <sys/errno.h>
 #include <sys/resource.h>
 #include <sys/queue.h>
+#include <bsd/stdlib.h>
 
 #include <ctype.h>
 #include <err.h>
--- freebsd-libs-9.0+ds1.orig/lib/libusbhid/usage.c
+++ freebsd-libs-9.0+ds1/lib/libusbhid/usage.c
@@ -36,6 +36,7 @@ __FBSDID("$FreeBSD$");
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <bsd/stdio.h>
 
 #include "usbhid.h"
 
--- freebsd-libs-9.0+ds1.orig/lib/libgeom/geom_ctl.c
+++ freebsd-libs-9.0+ds1/lib/libgeom/geom_ctl.c
@@ -39,6 +39,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <paths.h>
+#include <bsd/stdlib.h>
 
 #include <sys/queue.h>
 
--- freebsd-libs-9.0+ds1.orig/lib/libgeom/geom_getxml.c
+++ freebsd-libs-9.0+ds1/lib/libgeom/geom_getxml.c
@@ -33,6 +33,8 @@
 #include <sys/sysctl.h>
 #include <stdlib.h>
 #include <string.h>
+#include <bsd/stdlib.h>
+
 #include "libgeom.h"
 
 char *
--- freebsd-libs-9.0+ds1.orig/sys/netinet/libalias/alias_db.c
+++ freebsd-libs-9.0+ds1/sys/netinet/libalias/alias_db.c
@@ -157,6 +157,7 @@ __FBSDID("$FreeBSD$");
 #include <sys/errno.h>
 #include <sys/time.h>
 #include <unistd.h> 
+#include <bsd/stdlib.h>
 #endif
 
 #include <sys/socket.h>
